fr/basic/index.html

```diff
@@ -0,0 +1,995 @@
+        toute modification apportée au fichier dnas le prochain instantané à valider.
+        Toute modification qui ne sera pas ajoutée ne sera pas incluse à l'instantané
+        - cela signifie que vous pouvez créer vos instantanés de manière plus fine
+        que la plupart des systèmes de gestion de révisions.</p>
+
+      <p>Pour un exemple concret de la flexibilité de mise en attente une à une
+        de seulement certaines parties de fichiers modifiés, regardez l'option
+        '-p' de <code>git add</code> du livre Pro Git.</p>
+
+
+    </div>
+
+  </div>
+
+  <div class="box">
+    <h2>
+      <span class="docs">
+        <a target="new" href="http://git-scm.com/docs/git-status">docs</a> &nbsp;
+        <a target="new" href="http://git-scm.com/book/fr/Les-bases-de-Git-Enregistrer-des-modifications-dans-le-d%C3%A9p%C3%B4t#Vérifier-l%27état-des-fichiers">livre</a>
+      </span>
+      <a name="status">git status</a>
+      <span class="desc">voir le statut de vos fichiers du dossier courant et de la zone d'attente</span>
+    </h2>
+
+    <div class="block">
+      <p>Comme vous l'avez vu dans la section <code>git add</code>, pour voir
+        le statut de votre zone d'attente comparé au code de votre dossier de travail,
+        vous pouvez exécuter la commande <code>git status</code>. En utilisant
+        l'option <code>-s</code> l'affichage de la sortie sera réduit. Sans cette option
+        la commande <code>git status</code> vous donnera plus de contexte et d'indications.
+        Voici le même affichage de la sortie du statut avec et sans le <code>-s</code>.
+        La sortie courte ressemble à ça:
+      </p>
+
+<pre>
+<b>$ git status -s</b>
+<span class="green">A</span><span class="red">M</span> README
+<span class="green">A</span>  hello.rb
+</pre>
+
+      Alors que le même statut avec la sortie longue ressemble à ça:
+
+<pre>
+<b>$ git status</b>
+# On branch master
+#
+# Initial commit
+#
+# Changes to be committed:
+#   (use "git rm --cached &lt;file>..." to unstage)
+#
+# <span class="green">new file:   README</span>
+# <span class="green">new file:   hello.rb</span>
+#
+# Changed but not updated:
+#   (use "git add &lt;file>..." to update what will be committed)
+#   (use "git checkout -- &lt;file>..." to discard changes in working directory)
+#
+# <span class="red">modified:   README</span>
+#
+</pre>
+
+      <p>Vous pouvez rapidement vous apercevoir comme l'affichage court de la sortie
+        est beaucoup plus compact, mais la sortie longue a d'utiles informations et
+        indications dont vous pourriez avoir besoin.
+      </p>
+
+      <p>Git vous dira également quels fichiers ont été effacés
+        ou les fichiers modifiés ou mis en attente depuis votre dernière validation</p>
+<pre>
+<b>$ git status -s</b>
+<span class="green">M</span>  README
+ <span class="red">D</span> hello.rb
+</pre>
+
+        Vous pouvez voir qu'il y a deux colonnes dans la sortie courte du statut.
```

Comment:
Le "you can see" idiomatique en anglais est peu courant chez nous ; on traduira plus volontiers par "remarquez" que "vous pouvez voir".